### PR TITLE
Fixed array notation for required checkruns list

### PR DESCRIPTION
**Pull request by ricardoamador**
Fixed yaml format for checkruns list.

*List which issues are fixed by this PR. You must list at least one issue.*

*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].*

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Review by yusuf-goog — DISMISSED** [review]

File: autosubmit/flutter/autosubmit_main.yml [hunk 1]
```diff
@@ -17,5 +17,5 @@ approval_group: flutter-hackers
 run_ci: true
 support_no_review_revert: true
 required_checkruns:
-  “ci.yaml validation”
+  - “ci.yaml validation”
```
Comment (keyonghan) [inline autosubmit/flutter/autosubmit_main.yml:20]: Does the quote look right? shall it be `"` instead of `”`? Or it doesn't matter?

Comment (ricardoamador) [reply autosubmit/flutter/autosubmit_main.yml:20]: I think it just shows that way in githubs font. I used regular double quotes.

Comment (ricardoamador) [reply autosubmit/flutter/autosubmit_main.yml]: You are right!!!!! I just "fixed" them. I wonder what caused that?

**Review by yusuf-goog — APPROVED** [review]

File: autosubmit/flutter/autosubmit_main.yml (hunk 1 above)
Comment (keyonghan) [reply autosubmit/flutter/autosubmit_main.yml]: I used to hit it when directly copying from somewhere else.

**Pull request merged**